Assets/UI/WorldView/citybannermanager_CQUI_expansions.lua

```diff
@@ -206,17 +206,41 @@ function CityBanner.UpdateStats(self)
             self.m_Instance.CQUI_DistrictAvailable:SetHide(true);
             local pCityDistricts:table = pCity:GetDistricts();
             if (IsCQUI_SmartBanner_DistrictsEnabled()) then
+                local districtTooltipString = "";
+                local districtCount = 0;
+                local neighborhoodAdded = false;
                 -- Update the built districts 
                 for i, district in pCityDistricts:Members() do
                     local districtType = district:GetType();
                     local districtInfo:table = GameInfo.Districts[districtType];
                     local isBuilt = pCityDistricts:HasDistrict(districtInfo.Index, true);
+                    -- If the district is built, is not the City Center, is not a Wonder, and a Neighborhood Icon has already been shown...
                     if (isBuilt == true
-                        and districtInfo.DistrictType ~= "DISTRICT_WONDER"
-                        and districtInfo.DistrictType ~= "DISTRICT_CITY_CENTER") then
+                       and districtInfo.DistrictType ~= "DISTRICT_WONDER"
+                       and districtInfo.DistrictType ~= "DISTRICT_CITY_CENTER"
+                       and (districtInfo.DistrictType ~= "DISTRICT_NEIGHBORHOOD" or neighborhoodAdded == false)) then
                         SetDetailIcon(self.CQUI_DistrictBuiltIM:GetInstance(), "ICON_"..districtInfo.DistrictType);
+                        districtTooltipString = districtTooltipString .. "[ICON_".. districtInfo.DistrictType .. "] ".. Locale.Lookup(districtInfo.Name) .. "[NEWLINE]";
+                        districtCount = districtCount + 1;
+                        if (districtInfo.DistrictType == "DISTRICT_NEIGHBORHOOD") then
+                            neighborhoodAdded = true;
+                        end
                     end
                 end
+
+                -- Trim the trailing [NEWLINE]
+                districtTooltipString = string.sub(districtTooltipString, 1, string.len(districtTooltipString) - 9);
+
+                -- Determine the overlap of the district icons based on the number built
+                -- Note: GetNumZonedDistrictsRequiringPopulation does not include Aqueducts or Neighborhoods
+                -- The padding value was -12 before this dynamic calculation was introduced
+                local districtIconPadding = 8 + districtCount;
+                if districtIconPadding < 12 then districtIconPadding = 12; end
```

Comment:
Is there any reason we're using different padding calculations in the basegame vs expansions?

Reply:
yeah - the icons are a bit smaller in the basegame, so the 20 padding was way too tightly packed
-20 padding on Expansions:
![image](https://user-images.githubusercontent.com/8787640/89094860-5e9de400-d37d-11ea-9877-6d635ed3fa6b.png)

-20 padding on basegame:
![image](https://user-images.githubusercontent.com/8787640/89094907-c3593e80-d37d-11ea-94a6-ae7346121933.png)

Even -16 on basegame might be a bit too tight
![image](https://user-images.githubusercontent.com/8787640/89094960-277c0280-d37e-11ea-9c18-4de564328fd2.png)

Reply:
Actually, going to adjust for vanilla, the more I look at it the more I liked -14 over -16.  Slid the scale down accordingly.
![image](https://user-images.githubusercontent.com/8787640/89095038-c4d73680-d37e-11ea-8076-fd3de9ea8a65.png)

Reply:
That is quite unusual! I wonder why that's the case?

-14 looks quite good to me, while imo -16 would be the absolute most cramped that it could get while still being acceptable.

Reply:
If I'm picking things out of my XML correctly, the container hosting the icons on XP1/2 has a height of 41, while basegame has 34. 
@alexeyOnGitHub @alimulhuda @Infixo - any of y'all able to sign off?  (chaorace still has his banners are broken on Linux :/ )